sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -342,4 +361,11 @@ private static boolean isSonarLint(SensorContext context) {
     return context.getSonarQubeVersion().isGreaterThanOrEqual(SQ_VERSION_6_0) && context.runtime().getProduct() == SonarProduct.SONARLINT;
   }
 
+  private class CancellationException extends RuntimeException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make this a "static" inner class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS2694)